src/components/stepforms/CustomStepForm.vue

```diff
@@ -35,5 +35,14 @@ export default class CustomStepForm extends BaseStepForm<CustomStep> {
   get name(){
     return `Write a custom ${getTranslator(this.translator).label} query`;
   }
+
+  validate() {
+    const errors = this.$$super.validate();
+    if (errors !== null) {
+      return errors;
+    }
+    const translatorErrors = getTranslator(this.translator).validate({ ...this.editedStep })
+    return translatorErrors
```

Comment:
```suggestion 
   return translatorErrors;
```

Reply:
more generally, there a few missing semi-colons or extra spaces here and there, please use prettier :)